### PR TITLE
Cv 341 nav bar header footer

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -1,60 +1,37 @@
...
-                <a asp-controller="Home" asp-action="Index" class="govuk-header__link govuk-header__link--service-name">
-                    Commitments
-                </a>
-
-            </div>
-        </div>
-    </header>
-
+    <partial name="_Header" />
+    <partial name="_Menu" model="@accountHashedId" />
```
Comment (elliotritchie) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_Layout.cshtml]: Do/will we have any pages that won't have a `accountHashedId` route value? What happens here if the value is `null`?

Comment (Pauljgraham) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_Layout.cshtml]: The link points to the accounts url without an accountHashed. We really can't handle that here

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/EditDraftApprenticeship.cshtml [hunk 2]
```diff
@@ -16,10 +17,10 @@
             @Html.AntiForgeryToken()
             <partial name="DraftApprenticeship" for="@Model" />
             <button type="submit" class="govuk-button">Save</button>
-            @*<a class="govuk-button govuk-button--warning das-float-right" href="@Url.ProviderApprenticeshipServiceLink($"{Model.ProviderId}/apprentices/{Model.CohortReference}/{Model.DraftApprenticeshipHashedId}/delete")">Delete</a>
-        <p class="govuk-body  govuk-!-margin-bottom-0">
-            <a class="govuk-body govuk-link" href="@Url.ProviderApprenticeshipServiceLink($"{Model.ProviderId}/apprentices/{Model.CohortReference}/details")">Cancel</a>
-        </p>*@
+            <a class="govuk-button govuk-button--warning das-float-right" href="@LinkGenerator.CommitmentsLink($"accounts/{Model.AccountHashedId}/apprentices/{Model.CohortReference}/apprenticeships/{Model.DraftApprenticeshipHashedId}/delete")">Delete</a>
```
Comment (elliotritchie) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/EditDraftApprenticeship.cshtml]: Could an extension method be added for this URL too?

Comment (Pauljgraham) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/EditDraftApprenticeship.cshtml]: done

**Pull request merged**